spec/footer_spec.rb

```diff
@@ -1,4 +1,6 @@
 require_relative '../app/pages/page methods/mega-menu_met'
+require_relative '../app/assertions/footer_met'
+require_relative '../app/assertions/footer_assert'
 
 
```

Comment:
Add the name of regression suite between the single quotes in Describe call. In this case, the regression suite name is Footer. Ex: Describe 'Footer' do